cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/gomint/GoMintCloudNetBridgePlugin.java

```diff
@@ -0,0 +1,88 @@
+package de.dytanic.cloudnet.ext.bridge.gomint;
+
+import de.dytanic.cloudnet.driver.CloudNetDriver;
+import de.dytanic.cloudnet.ext.bridge.BridgeHelper;
+import de.dytanic.cloudnet.ext.bridge.BridgePlayerManager;
+import de.dytanic.cloudnet.ext.bridge.gomint.listener.GoMintCloudNetListener;
+import de.dytanic.cloudnet.ext.bridge.gomint.listener.GoMintPlayerListener;
+import de.dytanic.cloudnet.ext.bridge.listener.BridgeCustomChannelMessageListener;
+import de.dytanic.cloudnet.ext.bridge.player.IPlayerManager;
+import de.dytanic.cloudnet.ext.bridge.server.BridgeServerHelper;
+import de.dytanic.cloudnet.wrapper.Wrapper;
+import io.gomint.GoMint;
+import io.gomint.event.network.PingEvent;
+import io.gomint.plugin.Plugin;
+import io.gomint.plugin.PluginName;
+import io.gomint.plugin.Version;
+
+import java.util.concurrent.TimeUnit;
+
+@PluginName("CloudNet-Bridge")
+@Version(major = 1, minor = 2)
+public final class GoMintCloudNetBridgePlugin extends Plugin {
+
+    @Override
+    public void onInstall() {
+        GoMintCloudNetHelper.init();
+
+        CloudNetDriver.getInstance().getServicesRegistry().registerService(IPlayerManager.class, "BridgePlayerManager", new BridgePlayerManager());
+
+        this.initListeners();
+
+        Wrapper.getInstance().getTaskScheduler().schedule(BridgeHelper::updateServiceInfo);
+
+        this.runFirePingEvent();
+    }
+
+    @Override
+    public void onUninstall() {
+        CloudNetDriver.getInstance().getEventManager().unregisterListeners(this.getClass().getClassLoader());
+        Wrapper.getInstance().unregisterPacketListenersByClassLoader(this.getClass().getClassLoader());
+    }
+
+    private void initListeners() {
+        // GoMint API
+        super.registerListener(new GoMintPlayerListener(this));
+
+        // CloudNet
+        CloudNetDriver.getInstance().getEventManager().registerListener(new GoMintCloudNetListener());
+        CloudNetDriver.getInstance().getEventManager().registerListener(new BridgeCustomChannelMessageListener());
+    }
+
+    private void runFirePingEvent() {
+        super.getScheduler().schedule(() -> {
+            PingEvent pingEvent = new PingEvent(
+                    GoMintCloudNetHelper.getMotd(),
+                    GoMint.instance().getPlayers().size(),
+                    GoMintCloudNetHelper.getMaxPlayers()
+            );
+
+            boolean hasToUpdate = false, value = false;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=6226234)